src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Insights/Add-ArmApplicationInsightsAlertRuleWebhookAction.ps1

```diff
@@ -0,0 +1,26 @@
+function Add-ArmApplicationInsightsAlertRuleWebhookAction {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("ApplicationInsightsAlertRule")]
+    Param(
+        [PSTypeName("ApplicationInsightsAlertRule")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $ApplicationInsightsAlertRule,
+        [Parameter(Mandatory)]
+        [string]
+        $ServiceUri,
```

Comment:
Is this the webhook url? If so, we should rename to Webhook

Reply:
Yes it is. Renamed to `Webhook`